src/XMakeTasks/FileIO/WriteLinesToFile.cs

```diff
@@ -83,12 +85,12 @@ public override bool Execute()
                     }
                 }
 
-                Encoding encode = null;
+                Encoding encoding = s_defaultEncoding;
                 if (_encoding != null)
```

Comment:
This is *if* the user specified encoding (`_encoding` string set). I set it to the default so we could always safely use it rather than checking the parsed value for null because it failed to convert the string to an `Encoding` object.